sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/LogsHelperTests.cs

```diff
@@ -55,13 +55,11 @@ public void MessageIsSetToExceptionMessage(bool parseStateValues)
 
             Assert.Equal("Test Exception", message);
 
-            Assert.True(properties.TryGetValue("OriginalFormat", out string value));
-            Assert.Equal(log, value);
             Assert.True(properties.TryGetValue("name", out string name));
             Assert.Equal("tomato", name);
             Assert.True(properties.TryGetValue("price", out string price));
             Assert.Equal("2.99", price);
-            Assert.Equal(3, properties.Count);
+            Assert.Equal(2, properties.Count);
```

Comment:
I think this test is testing an incorrect behavior. In this test, the Exception message is what it expects as TraceTelemetry.Messsage, which is *incorrect*. Exception msg should be stored in ExceptionTelemetry only. 
Also, with this PR, in the case of exception occurring, the original format is fully lost.

The right fix would be to never populate Exception msg into TraceTelemetry's msg.

Reply:
The names are bit confusing here probably. It is not testing `TraceTelemetry` properties. `LogsHelper.GetMessageAndSetProperties` is a common method used for trace and exception telemetry. if LogRecord.Exception is not null we will export that as exception telemetry: https://github.com/Azure/azure-sdk-for-net/blob/e9695af3f4c26a3cdbf71063d6d80a75bd32ac92/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/LogsHelper.cs#L37-L52

Reply:
This covers the scenario when someone use log statement like below.


```
string log = "Hello from {name} {price}.";
try
{
    throw new Exception("Test Exception");
}
catch (Exception ex)
{
    logger.LogInformation(ex, log, "tomato", 2.99);
}
```

Reply:
> This covers the scenario when someone use log statement like below.
> 
> ```
> string log = "Hello from {name} {price}.";
> try
> {
>     throw new Exception("Test Exception");
> }
> catch (Exception ex)
> {
>     logger.LogInformation(ex, log, "tomato", 2.99);
> }
> ```

In this case, where would "Hello from {name} {price}" be stored?

Reply:
It will be dropped. I'm not sure if people use it in the same way as this one. We may need to gather feedback.

Reply:
Please modify changelog to call this out. This could be a breaking change for users who were relying on the formattedmsg or format itself to be present in the exceptiontelemetry. Changelog only mentions TraceTelemetry change, which is valid change, but questionable for ExceptionTelemetry, as we are waiting for feedback.

Reply:
After giving it some thought, I realized it doesn't make sense to only log the key-value pairs without including the original format. Since we're awaiting feedback, I'll leave the exception-handling part unchanged for now. I've pushed the new changes.